docs/pages/includes/device-spec.mdx

```diff
@@ -0,0 +1,35 @@
+```
```

Comment:
Should this be a YAML file? That way, we can also remove the opening/closing triple-backticks.

Reply:
From what I see, all other includes here are `.mdx` files containing a single codeblock so I followed existing convention. Should we do anything differently and switch them all out?

Reply:
Some docs pages include YAML files within `yaml` code snippets:

```
$ grep -RnE '\(![^!]+\.yaml!)' docs/pages | wc -l
28
```

It's not strictly necessary if the page renders correctly, but I do think it's a cleaner/more explicit approach if the partial consists only of a YAML document.